index.html

```diff
@@ -29,26 +30,25 @@
 
 <!-- Logo in Navigation Bar -->         
           <div class="logo"></div>
+
 <!-- End of Logo -->
 
           <ul>
-            <li><a href="https://www.google.com/?gws_rd=ssl">Home</a></li>
-            <li><a href="https://www.google.com/?gws_rd=ssl">Why VFix?</a></li>
-            <li><a href="https://www.google.com/?gws_rd=ssl">Services</a></li>
+            <li><a href="#home">Home</a></li>
+            <li><a href="#whyvfix">Why VFix?</a></li>
+            <li><a href="#services">Services</a></li>
           </ul>
         </nav><!-- End of Navigation Bar -->  
       </header><!-- End of Header Content -->
 
 <!-- Section 1 Tagline -->
-<section class="flexbox-grid">
-  <h1>ONSITE COMPUTER SUPPORT</h1>
+<a name="home"><section class="flexbox-grid">
+  </a><h1 class="tagline">ONSITE COMPUTER SUPPORT</h1>
```

Comment:
This formatting is a little tough to read, and I think it is invalid HTML (the `a` tag is not following proper nesting since it wraps only the opening `section` tag.